Dfe.Academies.Academisation.Domain.Core/ProjectAggregate/ProjectDetails.cs

```diff
@@ -1,111 +1,217 @@
er.MemberOfParliamentName, StringComparison.InvariantCultureIgnoreCase) && GeneralInformationSectionComplete == other.GeneralInformationSectionComplete && string.Equals(SchoolPerformanceAdditionalInformation, other.SchoolPerformanceAdditionalInformation, StringComparison.InvariantCultureIgnoreCase) && string.Equals(RationaleForProject, other.RationaleForProject, StringComparison.InvariantCultureIgnoreCase) && string.Equals(RationaleForTrust, other.RationaleForTrust, StringComparison.InvariantCultureIgnoreCase) && RationaleSectionComplete == other.RationaleSectionComplete && string.Equals(RisksAndIssues, other.RisksAndIssues, StringComparison.InvariantCultureIgnoreCase) && string.Equals(EqualitiesImpactAssessmentConsidered, other.EqualitiesImpactAssessmentConsidered, StringComparison.InvariantCultureIgnoreCase) && RisksAndIssuesSectionComplete == other.RisksAndIssuesSectionComplete && GoverningBodyResolution == other.GoverningBodyResolution && Consultation == other.Consultation && DiocesanConsent == other.DiocesanConsent && FoundationConsent == other.FoundationConsent && LegalRequirementsSectionComplete == other.LegalRequirementsSectionComplete && Nullable.Equals(EndOfCurrentFinancialYear, other.EndOfCurrentFinancialYear) && Nullable.Equals(EndOfNextFinancialYear, other.EndOfNextFinancialYear) && RevenueCarryForwardAtEndMarchCurrentYear == other.RevenueCarryForwardAtEndMarchCurrentYear && ProjectedRevenueBalanceAtEndMarchNextYear == other.ProjectedRevenueBalanceAtEndMarchNextYear && CapitalCarryForwardAtEndMarchCurrentYear == other.CapitalCarryForwardAtEndMarchCurrentYear && CapitalCarryForwardAtEndMarchNextYear == other.CapitalCarryForwardAtEndMarchNextYear && string.Equals(SchoolBudgetInformationAdditionalInformation, other.SchoolBudgetInformationAdditionalInformation, StringComparison.InvariantCultureIgnoreCase) && SchoolBudgetInformationSectionComplete == other.SchoolBudgetInformationSectionComplete && YearOneProjectedCapacity == other.YearOneProjectedCapacity && YearOneProjectedPupilNumbers == other.YearOneProjectedPupilNumbers && YearTwoProjectedCapacity == other.YearTwoProjectedCapacity && YearTwoProjectedPupilNumbers == other.YearTwoProjectedPupilNumbers && YearThreeProjectedCapacity == other.YearThreeProjectedCapacity && YearThreeProjectedPupilNumbers == other.YearThreeProjectedPupilNumbers && string.Equals(SchoolPupilForecastsAdditionalInformation, other.SchoolPupilForecastsAdditionalInformation, StringComparison.InvariantCultureIgnoreCase) && string.Equals(KeyStage2PerformanceAdditionalInformation, other.KeyStage2PerformanceAdditionalInformation, StringComparison.InvariantCultureIgnoreCase) && string.Equals(KeyStage4PerformanceAdditionalInformation, other.KeyStage4PerformanceAdditionalInformation, StringComparison.InvariantCultureIgnoreCase) && string.Equals(KeyStage5PerformanceAdditionalInformation, other.KeyStage5PerformanceAdditionalInformation, StringComparison.InvariantCultureIgnoreCase) && Equals(AssignedUser, other.AssignedUser);
```

Comment:
This seems a bit difficult to manage.

Reply:
Pushed the reformatted version - can't think of a way to make the code any less horrific, though.